masterlist.yaml

```diff
@@ -1003,7 +1003,7 @@ globals:
   - <<: *versionXIncY
     subs:
       - '**SKSE Plugin ([ENB Helper SE](https://www.nexusmods.com/skyrimspecialedition/mods/23174/))**'
-      - '[ENBSeries](http://enbdev.com/download_mod_tesskyrimse.html)'
+      - '[ENBSeries](https://enbdev.com/download_mod_tesskyrimse.html)'
```

Comment:
The ENB site doesn't seem to support HTTPS. Also, that link is outdated. It's `http://enbdev.com/download_mod_tesskyrimse.htm` now.

Reply:
I have no problem opening that website using `https`, I checked it before changing it.
Besides that, the url can be updated of course.

Reply:
It works, it's just technically not correct.

Reply:
I guess you're right. Will change it in another PR.